lib/types/api.types.ts

```diff
@@ -78,10 +78,14 @@ export interface LocationAPI {
 
 /* Parameters API */
 
-export interface ParametersAPI {
-  installation: KeyValueMap
-  instance: KeyValueMap
-  invocation?: SerializedJSONValue
+export interface ParametersAPI<
+  TInstallationParameters extends KeyValueMap,
```

Comment:
If 'T' here stands for 'Type' then we don't do it usually.

Reply:
Okay i can drop the T = Type naming convention 👍

Reply:
done.